resolwe_bio/tools/go_genesets.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python2
+"""Parse GAF file to genesets."""
+import argparse
+import csv
+import json
+
+from collections import defaultdict
+
+from resolwe_runtime_utils import export  # pylint: disable=import-error
+from utils import gzopen  # pylint: disable=import-error
+
+
+def parse_arguments():
+    """Parse command line arguments."""
+    parser = argparse.ArgumentParser(description="Parse Diff Exp output files")
+    parser.add_argument('gaf_file', help="GAF file.")
+    parser.add_argument('--source', help="GeneID source.")
```

Comment:
This field is required in `'upload-geneset` process, so it cannot be optional here.